Libs/MRML/Widgets/qMRMLSliceView.h

```diff
@@ -84,19 +84,19 @@ class QMRML_WIDGETS_EXPORT qMRMLSliceView : public ctkVTKSliceView
 
   /// Convert device coordinates to XYZ coordinates. The x and y
   /// components of the return value are the positions within a
-  /// LightBox pane and the z component of the return value (rounded
-  /// to the nearest integer) is the pane in the LightBox
+  /// slice view pane and the z component of the return value is always 0.0. It
+  /// used to be the pane (rounded to the nearest integer) in the LightBox
```

Comment:
Same remark as for the VTK level, would it make sense to introduce convertRASToXY API instead and deprecate this one?